php/php.editor/test/unit/data/testfiles/index/testPHP80ConstructorPropertyPromotion/testPHP80ConstructorPropertyPromotion.php.indexed

```diff
@@ -2,7 +2,7 @@
 
 Document 0
 Searchable Keys:
-  clz : #anon#testphp80constructorpropertypromotion_php#1;#anon#testPHP80ConstructorPropertyPromotion_php#1;2162;;;;1;;0;<TESTURL>/testPHP80ConstructorPropertyPromotion.php;;
+  clz : #anon#testphp80constructorpropertypromotion_php#1;#anon#testPHP80ConstructorPropertyPromotion_php#1;2166;;;;1;;0;<TESTURL>/testPHP80ConstructorPropertyPromotion.php;;
```

Comment:
By this change:
https://github.com/apache/netbeans/blob/9a45ad273f06d392ce724b92966dff6b7e73cbb8/php/php.editor/src/org/netbeans/modules/php/editor/model/nodes/ClassInstanceCreationInfo.java#L56-L65